examples/snippets/filesystem/cloud/s3_download.py

```diff
@@ -0,0 +1,17 @@
+"""
+An example script to demonstrate the use of the s3_download function to download file from.
+"""
+
+import tamr_toolbox as tbox
+from boto3.session import Session
+
+S3Client = Session("credentials").client("s3")
+
+# download file from AWS S3
+# download a file on GCS "s3://my-bucket/path-to-file" to "my_local_directory/my_file.txt"
+tbox.filesystem.cloud.s3_download(
+    cloud_client=S3Client,
+    source_filepath="s3://my-bucket/path-to-file",
```

Comment:
My testing shows that the `source_filepath` should not include `s3://<bucket-name>` and should just be the component of the filepath following this.
```suggestion
    source_filepath="path-to-file",
```